api/config.yaml

```diff
@@ -53,6 +53,15 @@ components:
           $ref: "#/components/schemas/OpenSearch"
         signalFx:
           $ref: "#/components/schemas/SignalFX"
+    UpdateDataStoreConfigRequest:
+      type: object
+      properties:
+        dataStores:
+          type: array
+          items:
+            $ref: "#/components/schemas/DataStore"
+        defaultDataStore:
+          $ref: "#/components/schemas/SupportedDataStores"
```

Comment:
I think this should be a string instead. The user might have 2 different jaeger instances and want to switch between them. This should reference the data store id

Reply:
Great callout  @mathnogueira in that case, should we include a name entry or something like that?